### PR TITLE
Skip alredy downloaded file during uncompress

### PR DESCRIPTION
**Pull request by linhandev**
See if there is a file with the same filename and size in output dir. If so, skip this download

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 pre-commit
 scikit-image
 numpy
-paddlepaddle-gpu>=2.2.0
```
Comment (parap1uie-s) [inline requirements.txt:4]: 为什么要去掉paddle？

Comment (linhandev) [reply requirements.txt:4]: 啊这笔误了😓我这cpu环境，装依赖的时候暂时删了，忘加回来了。

不过感觉这块也可以不写paddlepaddle-gpu，seg就没写。装gpu版本还得装cuda，gpu的paddle针对不同的cuda版本好像有不同的pip。感觉可以在readme里用一个section写一下怎么装依赖，先requirements.txt，之后给一个paddle官网安装教程的链接

Comment (shiyutang) [reply requirements.txt:4]: 嗯嗯，也可以

**Comment by linhandev** [comment]
有空可以看一下，需要的地方我可以改。时间长了容易出confilict

**Comment by shiyutang** [comment]
> 

嗯嗯，以为你要重新cherry-pick再提交

File: requirements.txt [hunk 2]
```diff
@@ -10,3 +9,4 @@ visualdl
 sklearn
 filelock
 nibabel
+opencv-python
```
Comment (shiyutang) [inline requirements.txt:12]: opencv-python is not needed

**Review by shiyutang — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (linhandev) [reply requirements.txt:12]: image

The last one is for debug, but I think it's used in visualization

Comment (shiyutang) [reply requirements.txt:12]: You may want to pull the repo, I have deleted the visualize.py

**Review by shiyutang — CHANGES_REQUESTED** [review]
There is a conflict that needs to be resolved on your computer.

**Comment by shiyutang** [comment]
The content of this PR has been incorporated into the current code.

**Pull request closed without merging**